cmd/tasks/claim.go

```diff
@@ -0,0 +1,94 @@
+package tasks
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	"github.com/resonatehq/resonate/pkg/client"
+	v1 "github.com/resonatehq/resonate/pkg/client/v1"
+	"github.com/spf13/cobra"
+)
+
+// Example command usage for claiming a task
+var claimTasksExample = `
+# Claim a task 
+resonate tasks claim --id foo --counter 1 --process-id bar --ttl 1m`
+
+// ClaimTaskCmd returns a cobra command for claiming a task.
+func ClaimTaskCmd(c client.Client) *cobra.Command {
+	var (
+		id        string        // Task ID to claim
+		counter   int           // Counter for the task claim
+		processId string        // Unique process ID identifying the claimer
+		ttl       time.Duration // Time to live for the task claim
+		requestId string        // Unique tracking ID for the request
+	)
+
+	// Define the cobra command
+	cmd := &cobra.Command{
+		Use:     "claim",
+		Short:   "Claim a task",
+		Example: claimTasksExample,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// Validate required flags
+			if id == "" {
+				return errors.New("id is required")
+			}
+			if counter <= 0 {
+				return errors.New("counter is required")
+			}
+			if processId == "" {
+				return errors.New("process-id is required")
+			}
+
+			// Create parameters for the claim task request
+			params := &v1.ClaimTaskParams{
+				RequestId: &requestId, // Optional tracking ID
+			}
+
+			// Create the body for the claim task request
+			body := v1.ClaimTaskJSONRequestBody{
+				Id:        id,
+				Counter:   counter,
+				ProcessId: processId,
+				Ttl:       int64(ttl.Milliseconds()), // Convert duration to milliseconds
```

Comment:
I believe `ttl.Milliseconds()` is already an int64 so we shouldn't have to convert it